test/TestCase.cpp

```diff
@@ -18,11 +18,12 @@
 #include <test/Common.h>
 #include <test/TestCase.h>
 
-#include <boost/algorithm/string/predicate.hpp>
 #include <boost/algorithm/string.hpp>
+#include <boost/algorithm/string/predicate.hpp>
+#include <libsolutil/AnsiColorized.h>
```

Comment:
You need to move this line upper, for more info look here:
https://github.com/ethereum/solidity/blob/develop/CODING_STYLE.md#12-include-headers

Reply:
@mijovic Ok, It's done :)